<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputHandler</class>
 <widget class="QWidget" name="InputHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="lo_main">
   <item>
    <layout class="QHBoxLayout" name="__lo_btns">
     <item>
      <layout class="QHBoxLayout" name="lo_weight"/>
     </item>
     <item>
      <widget class="QPushButton" name="bt_addNew">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_browse">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="shortcut">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_clear">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="shortcut">
        <string notr="true"/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scra">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="lo_scroll">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bt_addNew</tabstop>
  <tabstop>bt_browse</tabstop>
  <tabstop>bt_clear</tabstop>
  <tabstop>scra</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
